python/cuml/manifold/t_sne.pyx

```diff
@@ -413,15 +419,16 @@ class TSNE(Base):
             self.Y = None
 
     def fit_transform(self, X):
-        """Fit X into an embedded space and return that transformed output.
+        """
+        Fit X into an embedded space and return that transformed output.
```

Comment:
I think that leaving the one-line intro on the same line as """ is the norm (the way you had it previously):

https://numpydoc.readthedocs.io/en/latest/example.html#example

Reply:
Oh ok :)